src/rocket.py

```diff
@@ -0,0 +1,19 @@
+import pygame
+
+class Rocket(pygame.sprite.Sprite):
+    def __init__(self, x, y, width, height, color, velocity, acceleration):
+        pygame.sprite.Sprite.__init__(self)
+        self.image = pygame.Surface((width, height))
+        self.image.fill(color)
+        self.rect = self.image.get_rect()
+        self.rect.x = x
+        self.rect.y = y
+        self.velocity = velocity
+        self.acceleration = acceleration
+
+    def move(self):
+        self.rect.x += self.velocity
+        self.velocity += self.acceleration
```

Comment:
```suggestion
```
move rect only horizontally? Independet from timestep?

Reply:
I would prefer to have an extra position attribute and update rect accordingly. I feel the decoupling of physics and updating game mechanics is better then